readme.md

```diff
@@ -260,6 +260,41 @@ const validate = (input: unknown) => {
 };
 ```
 
+### Boolean(val) is now also a predicate
+
+```ts
+import "@total-typescript/ts-reset/boolean-predicate";
+```
+
+When you're using `Boolean(val)`, Typescript doesn't understand it as a predicate on `val`.
+
+```ts
+// BEFORE
+
+const getUserName = (user?: User) => {
+  const isUser = Boolean(user);
+  if (isUser) {
+    console.log(user.name); // error: user is possibly undefined
+  }
+};
+```
+
+With this rule enabled, `Boolean(val)` is now also a type predicate. This means you can use it to narrow types:
+
+```ts
+// AFTER
+import "@total-typescript/ts-reset/is-array";
```

Comment:
```suggestion
import "@total-typescript/ts-reset/boolean-predicate";
```

Noticed this in [Matt's tweet](https://twitter.com/mattpocockuk/status/1633504795079188487).